app/services/scenes/scenes.ts

```diff
@@ -1,31 +1,31 @@
 import Vue from 'vue';
+import { uniqBy } from 'lodash';
+import { Subject } from 'rxjs';
+import uuid from 'uuid/v4';
 import { without } from 'lodash';
-import { StatefulService, mutation } from 'services/stateful-service';
+import { mutation, StatefulService } from 'services/stateful-service';
 import { TransitionsService } from 'services/transitions';
 import { WindowsService } from 'services/windows';
 import {
   IScene,
-  Scene,
+  ISceneCreateOptions,
   ISceneItem,
-  SceneItem,
+  IScenesServiceApi,
   IScenesState,
-  ISceneCreateOptions,
-  IScenesServiceApi
+  Scene,
+  SceneItem,
```

Comment:
not sure why these imports change order? it just seems like dirtying the diff for no actual reason

Reply:
Changes of this file have to be excluded from this PR

Reply:
How do you mean? We need the scene dedup that's in here.